jpegenc/models/rgb2ycbcr/commons.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/python
```

Comment:
I realize this was derived from the code in the repo but some of this needs to be revisited.  

I don't think the constants `*ACTIVE*` have much use nor `sintbv`.  I would like to see these removed and this file renamed to rounding (and only have the rounding functions).

The `*ACTIVE*` consts can be removed because the `reset` already has an `active` attribute that can be used and other `bool` signals make more sense to use the `True` and `False` versus using, the truth values are more readable than the *hardware* levels.

```python
    enable.next = True
    disable.next = True
```

The above is more readable than `enable.next = ACTIVE_HIGH`.